torch_geometric/nn/conv/gcn_conv.py

```diff
@@ -17,21 +17,22 @@
 
 @torch.jit._overload
 def gcn_norm(edge_index, edge_weight=None, num_nodes=None, improved=False,
-             add_self_loops=True, dtype=None):
-    # type: (Tensor, OptTensor, Optional[int], bool, bool, Optional[int]) -> PairTensor  # noqa
+             add_self_loops=True, flow="source_to_target", dtype=None):
+    # type: (Tensor, OptTensor, Optional[int], bool, bool, str, Optional[int]) -> PairTensor  # noqa
```

Comment:
```suggestion
    # type: (Tensor, OptTensor, Optional[int], bool, bool, Optional[str], Optional[int]) -> PairTensor  # noqa
```

Reply:
Optional[str] = Union[str, None], which is not what we want here I believe (passing `flow=None` should be wrong). It does not necessarily mean optional argument. Just like `add_self_loops=True`, the signature is still just `bool`, not `Optional[bool]`.